core/src/main/scala/kafka/server/DynamicConfigManager.scala

```diff
@@ -71,9 +71,11 @@ object ConfigEntityName {
  *
  * This will fire a watcher on all brokers. This watcher works as follows. It reads all the config change notifications.
  * It keeps track of the highest config change suffix number it has applied previously. For any previously applied change it finds
- * it checks if this notification is larger than a static expiration time (say 10mins) and if so it deletes this notification.
+ * it checks if this notification is larger than a static expiration time (default to 15 mins) and if so it deletes this notification.
  * For any new changes it reads the new configuration, combines it with the defaults, and updates the existing config.
  *
+ * See [[kafka.common.ZkNodeChangeNotificationListener]] for details
```

Comment:
In https://github.com/apache/kafka/pull/679, we make DynamicConfigManager to use the ZkNodeChangeNotificationListener. Add a link here.